### PR TITLE
Abuse filter settings on Elements wiki per T1234

### PR DESCRIPTION
**Pull request by DeltaQuad-DQ**

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -136,6 +136,30 @@
 		'default' => false,
 		'metawiki' => true,
 	),
+	'wgAbuseFilterAvailableActions' => array(
+		'default' => array(
+			 'flag' => true,
```
Comment (Reception123) [inline LocalSettings.php]: please fix the tabbing here

Comment (DeltaQuad-DQ) [inline LocalSettings.php]: @Reception123 what do you mean?

Comment (Reception123) [inline LocalSettings.php]: 'flag' is tabbed wrongly it is ahead of the others. like
 'flag' instead of
'flag'

Comment (DeltaQuad-DQ) [inline LocalSettings.php]: Fixed

**Pull request merged**